uap_tracker/tracker.py

```diff
@@ -43,34 +44,43 @@ def update(self, frame, validate_target=True):
                 stationary_check_thold = 4
                 stationary_check_max = 5
 
-                if len(self.bboxes) > 10:
-                    if self.tracking_state == Tracker.PROVISIONARY_TARGET:
-                        self.tracking_state = Tracker.ACTIVE_TARGET
+                # MG: Default to yellow as it needs to prove itself as a target first
+                if len(self.bboxes) <= 10:
+                    # self.bbox_color = (25, 175, 175)
+                    self.tracking_state = Tracker.PROVISIONARY_TARGET
```

Comment:
Not sure this branch is necessary as tracking_state is already initialised to PROVISIONARY

Reply:
Yeah, there was a bug in that tracking code to be fair. The more I think about it the more I think this part of the code is very very important as it has such an impact on performance. I want to spend some more time on this and try and improve the implementation.

Reply:
Yup, my bad on the bug, thanks for fixing it. Please can you remove the dead code.